src/content/guides/integrating-third-party-asset.mdx

```diff
@@ -0,0 +1,353 @@
+| authorization  | String                                              | required  | A reference to the asset of the party paying the Payment Request.                                                                   |
+| merchantName   | String                                              | required  | The name of the merchant who created the Payment Request.                                                                           |
+| merchantId     | String                                              | required  | Your identifier for the merchant receiving payment.                                                                                 |
+| idempotencyKey | String                                              | required  | A unique value that can be used to recognise subsequent retries of the same request.                                                |
+| transactionId  | String                                              | required  | An ID for the transaction in Centrapay’s system.                                                                                    |
+| status         | String                                              | required  | The status of the asset transaction. See [possible status values](#statuses).         |
+| type           | String                                              | required  | The type of transaction. Possible values are payment or refund.                                                                     |
+| failureReason  | String                                              | optional  | Required if the status is failed. See [possible failure reasons](#failure-reasons).     |
+| refundable     | Boolean                                             | optional  | Required if type is payment and status is successful. A flag indicating whether a payment is refundable.                            |
+| refundBefore   | [Timestap](https://docs.centrapay.com/api/data-types#timestamp) | optional  | The latest time at which a refund can be initiated.                                                                                 |
+
+### Statuses
+
+| Name | Description |
+| --- | --- |
+| pending | The transaction is processing.  |
+| successful | The transaction has been successfully processed. |
+| failed | The transaction has been unable to be successfully processed. A failure reason is expected to be provided when status is failed. |
+
+### Failure Reasons
+
+See endpoint spec for possible failure reasons.
+
+### Pay Endpoint
+
+This endpoint is used to initiate payment. If payment is completed asynchronously, the status will be `pending` and the [get transaction endpoint](https://www.notion.so/Integrating-a-third-party-asset-v2-523b91cdea034dc08187b63341f8f1e4) will be used to poll until the status is `successful` or `failed`.
+
+```mermaid
+sequenceDiagram
+	participant P as Patron
```

Comment:
@tom-blake I knew we would regret using single letter variables in the first mermaid example

Reply:
Is there a preference for full or shortened names?